test/e2e/packetcapture_test.go

```diff
@@ -151,24 +151,30 @@ func packetCaptureHostPublicKey(pubKey []byte) packetCaptureOption {
 	}
 }
 
-func getPacketCaptureCR(name string, namespace string, clientPodName string, destinationPodName string, sftpURL string, packet *crdv1alpha1.Packet, direction crdv1alpha1.CaptureDirection, options ...packetCaptureOption) *crdv1alpha1.PacketCapture {
+func packetCaptureSourcePod(namespace, clientPodName string) packetCaptureOption {
```

Comment:
s/clientPodName/sourcePodName

for consistency

or maybe just use `name` as the parameter name for both `packetCaptureSourcePod` and `packetCaptureDestinationPod`